packages/SwingSet/src/types-external.js

```diff
@@ -222,8 +222,9 @@ export {};
  *                                 vatSyscallHandler: unknown) => Promise<VatManager>,
  *            } } VatManagerFactory
  * @typedef { { deliver: (delivery: VatDeliveryObject) => Promise<VatDeliveryResult>,
+ *              replayOneDelivery: (delivery: VatDeliveryObject, expectedSyscalls: VatSyscallObject[], deliveryNum: number) => Promise<VatDeliveryResult>,
  *              replayTranscript: (startPos: StreamPosition | undefined) => Promise<number?>,
- *              makeSnapshot?: (ss: SnapStore) => Promise<SnapshotInfo>,
+ *              makeSnapshot: (ss: SnapStore) => Promise<SnapshotInfo>,
```

Comment:
While this is a type change only, it was a misleading one. Conditionality of methods like that is really hard to express and flow through in types, and in this case, there was a typing mistake in `getManager` that confused me. I'll revert this and fix `getManager`'s type to avoid spreading the confusion, but it would be great to find a better way to express the manager variants.